nbtc_swap/sources/nbtc_swap.move

```diff
@@ -23,7 +25,8 @@ public struct Vault has key, store {
     id: UID,
     nbtc_balance: Balance<NBTC>,
     sui_balance: Balance<SUI>,
-    price_per_nbtc_satoshi_in_mist: u64,
+    /// 1 nano nBTC price in MIST
+    nnbtc_price: u64,
```

Comment:
I am not sure about the renaming. It nbtc_satoshi_price not nano nbtc. one nano btc is 0.1 satoshi

Reply:
thanks, updated.